nf_core/lint/nextflow_config.py

```diff
@@ -229,10 +229,14 @@ def nextflow_config(self):
 
     # Check that the DAG filename ends in ``.svg``
     if "dag.file" in self.nf_config:
-        if self.nf_config["dag.file"].strip("'\"").endswith(".svg"):
-            passed.append("Config ``dag.file`` ended with ``.svg``")
+        _, dag_file_suffix = os.path.splitext(self.nf_config["dag.file"].strip("'\""))
```

Comment:
Not sure at all about `_, `

Reply:
Would you prefer `dag_file_suffix = os.path.splitext(self.nf_config["dag.file"].strip("'\""))[1]`?

Reply:
not familiar at all with Python, so highly confused by this syntax

Reply:
For Python, `_` is a normal variable name, but by convention it is a throwaway. What happens is a the tuple returned by the splitext function (split away the file extension), which consists of the last extension and the rest of the filename (e.g. "file.tar.gz" is split into "file.tar" and ".gz" and returned as a Python tuple `("file.tar", ".gz")`) is unpacked into two variables, one is `_` and the other is `dag_file_suffix`. Using indexes hurts my eyes and tuple unpacking pleases them, so that's what I do ;-)

Reply:
The additional "benefit" of the tuple unpacking in this variety is, that is needs to be precise with what it expects. In this case it needs to get a tuple with exactly two elements. If there is no extension, the `os.path.splitext` function will still return a tuple, but the extension will be an empty string which is not in the allowed extensions and thus the test will fail as expected.

I hope this helps you be comfortable with the proposed code change?

Reply:
No worries about me, I'm happy with the improvment

Reply:
Just need to fix the CHANGELOG, and we're good to go